sql/expression/function/json_extract_test.go

```diff
@@ -47,6 +47,8 @@ func TestJSONExtract(t *testing.T) {
 		row      sql.Row
 		expected interface{}
 	}{
+		{f2, sql.Row{nil, "$.b.c"}, nil},
+		{f2, sql.Row{json, "$.foo"}, nil},
```

Comment:
Maybe we can add `{f2, sql.Row{json, "FOO"}, nil},` too

Reply:
This is going to be a error, I am adding it as it